sdks/python/apache_beam/examples/inference/sklearn_japanese_housing_regression.py

```diff
@@ -0,0 +1,165 @@
+  for i, model in enumerate(MODELS):
+    required_features = dataframe[model['required_features']]
+    if not required_features.isnull().any().any():
+      return i
+  return -1
+
+
+class LoadDataframe(beam.DoFn):
+  def process(self, file_name: str) -> Iterable[pandas.DataFrame]:
+    """ Loads data files as a pandas dataframe."""
+    file = FileSystems.open(file_name, 'rb')
+    dataframe = pandas.read_csv(file)
+    for i in range(dataframe.shape[0]):
+      yield dataframe.iloc[[i]]
+
+
+def report_predictions(prediction_result):
+  true_result = prediction_result.example['TradePrice'].values[0]
+  inference = prediction_result.inference
+  return 'True Price %.0f, Predicted Price %.0f' % (true_result, inference)
+
+
+def parse_known_args(argv):
+  """Parses args for the workflow."""
+  parser = argparse.ArgumentParser()
+  parser.add_argument(
+      '--input',
+      dest='input',
+      required=True,
+      help='A single or comma separated list of files or uris.')
+  parser.add_argument(
+      '--model_path',
+      dest='model_path',
+      required=True,
+      help='A path from where all models can be read.')
+  parser.add_argument(
+      '--output',
+      dest='output',
+      required=True,
+      help='Path to save output predictions.')
+  return parser.parse_known_args(argv)
+
+
+def inference_transform(model_name, model_path):
+  # These sklearn models are a pipeline that use pandas.
+  model_filename = model_path + model_name + '.pickle'
+  model_loader = SklearnModelHandlerPandas(
+      model_file_type=ModelFileType.PICKLE, model_uri=model_filename)
+  transform_name = 'RunInference ' + model_name
+  return transform_name >> RunInference(model_loader)
+
+
+def run(argv=None, save_main_session=True):
+  """Entry point. Defines and runs the pipeline."""
+  known_args, pipeline_args = parse_known_args(argv)
+  pipeline_options = PipelineOptions(pipeline_args)
+  pipeline_options.view_as(SetupOptions).save_main_session = save_main_session
+
+  with beam.Pipeline(options=pipeline_options) as p:
+    # This example uses a single file, but it is possible to use many files.
+    file_names = p | 'FileNames' >> beam.Create([known_args.input])
+    loaded_data = file_names | beam.ParDo(LoadDataframe())
+
+    # Some examples don't have all features. Pipelines
+    # that expect those fields will fail. There are many ways to deal with
+    # missing data. This example illustrates how to assign predictions to
+    # different models depending upon what data is available.
+    [all, floor_area, stations, no_features] = (
+        loaded_data
+        | 'Partition' >> beam.Partition(sort_by_features, len(MODELS)))
+
+    model_path = known_args.model_path
+    prediction_1 = all | inference_transform('all_features', model_path)
+    prediction_2 = floor_area | inference_transform('floor_area', model_path)
+    prediction_3 = stations | inference_transform('stations', model_path)
```

Comment:
This model couldn't be found in the apache-beam-ml bucket. I can see the rest three and `frontage.pickle`

Reply:
yeah, that was a mistake, I've added the model.